test_conformance/device_execution/enqueue_flags.cpp

```diff
@@ -54,7 +54,7 @@ static const char* enqueue_flags_wait_kernel_simple[] =
     NL, ""
     NL, "  if((index + 1) == BITS_DEPTH)"
     NL, "  {"
-    NL, "    barrier(CLK_LOCAL_MEM_FENCE);"
+    NL, "    barrier(CLK_GLOBAL_MEM_FENCE);"
```

Comment:
Hmm, I'm guessing you didn't intend these changes to be part of the PR, did you?

Reply:
Looks like there was some diff between Khronos master and my fork which led to these unnecessary changes. I have merged Khronos:master to my local fork.
Let me know if anything else is messed up.